setup.py

```diff
@@ -71,9 +73,11 @@ def run(self):
         'git://github.com/NeuralEnsemble/libNeuroML.git#egg=libNeuroML',
         'git://github.com/zopefoundation/ZODB.git#egg=ZODB',
         'git://github.com/RDFLib/FuXi#egg=FuXi-1.4.dev-r0',
+        'git+https://github.com/RDFLib/rdflib-zodb#egg=rdflib-zodb-1.1-dev',
+        'git+https://github.com/mwatts15/YAROM@dev#egg=yarom-0.7.1',
     ],
     version = '0.5.3',
-    packages = ['PyOpenWorm'],
+    packages = ['PyOpenWorm', 'PyOpenWorm.v0', 'PyOpenWorm.graphObjectAdapter'],
```

Comment:
To focus the concern, the extra packages are cruft.  They add more work to maintain for everyone else, even if they simplify your process.  They are basically there for your benefit, but they don't help clarify PyOpenWorm.  They are a side effect of your work; fine for your own local environment, but not great for a public code base where other people need to understand what is going on as easily as they can.